### ISSUE TITLE
 Join the Backdrop contributor community?

### ISSUE THREAD

**Issue by nico8948** [backdrop-ops/contrib#798]
Can i join the Backdrop contributor community?

**Comment by laryn** [comment]
@nico8948 Of course, and we'd love to have you! The README linked below contains further information about the process to officially join:

https://github.com/backdrop-ops/contrib/blob/main/README.md

**Comment by stpaultim** [comment]
@nico8948 Any questions we can help you with?

**Comment by nico8948** [comment]
i've forked    https://github.com/backdrop-contrib/printmpdf,    updated it do the latest   php version en mpdf.    i've made a merge request.  No answer?

Maybe the wrong git site? https://gitlab.com/ulterius/salestracker-crm      It is a Sales backdrop module, is this interesting for Backdrop?

**Comment by yorkshire-pudding** [comment]
> i've forked https://github.com/backdrop-contrib/printmpdf, updated it do the latest php version en mpdf. i've made a merge request. No answer?

@nico8948 - you haven't added `Fixes #5` to the top of the pull request description; that would link the PR to the issue. Secondly, the issue is not very clear that you are updating the library and making it compatible with PHP 8.2 so it would be hard for anyone to realise that you wanted to merge it from the issue.

Once you've done the above, and tagged the maintainer (see the README) in the issue and waited 2 weeks, you can then call on the Bug squad by tagging `@backdrop-contrib/bug-squad` in the issue.
